src/components/List/ListItem.tsx

```diff
@@ -132,11 +138,28 @@ class ListItem extends UIComponent<ReactProps<ListItemProps>> {
         headerMediaCSS={styles.headerMedia}
         contentCSS={styles.content}
         onClick={this.handleClick}
-        {...accessibility.attributes.root}
-        {...accessibility.keyHandlers.root}
+        {...accessibility.attributes.item}
+        {...accessibility.keyHandlers.item}
         {...rest}
+        {...!wrapper && { onClick: this.handleClick }}
       />
     )
+
+    if (wrapper) {
+      return Slot.create(wrapper, {
+        defaultProps: {
+          className: cx('ui-list__item__wrapper', classes.wrapper),
```

Comment:
Consider reference to ListItem.className here as well opposed to hard coded string.